data/cleansed/countries.json

```diff
@@ -832,13 +832,6 @@
     "iso_3166_3": "KIR",
     "currency": "AUD"
   },
-  {
-    "name": "Kosovo",
-    "continent": "AS",
-    "iso_3166_2": "XK",
-    "iso_3166_3": "RKS",
-    "currency": "EUR"
-  },
```

Comment:
this looks like a revert of https://github.com/flowcommerce/json-reference/commit/9c50fd4584a05b4a64f6f5e2973a838e53ee0585, i think we need to keep it for backwards-compatibility